web/.eslintrc

```diff
@@ -1,51 +1,36 @@
 {
+  // default parser does not support stage 3 so use this
+  "parser": "@babel/eslint-parser",
   "extends": [
-    "eslint:recommended",
-    "plugin:react/recommended"
+    "airbnb",
+    "prettier"
```

Comment:
Pretty much just using prettier and airbnb as is. Couple of exceptions noted below.